### PR TITLE
Screens

### PR DESCRIPTION
**Pull request by tanveertkd**
Added screens for landing page, home and comments.  

Routes:  

- [Landing](https://deploy-preview-1--ti-social.netlify.app/)
- [Home](https://deploy-preview-1--ti-social.netlify.app/home)
- [Comments](https://deploy-preview-1--ti-social.netlify.app/post)
- [Profile](https://deploy-preview-1--ti-social.netlify.app/profile)

You can click on the 'Go to Home' button on the landing page to go into the home page.   
Clicking on the comment button will take you to the post page.  
Clicking on the user on the left sidebar will take you to the profile page

### CODE REVIEW

File: src/components/NavBar/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                {/*  Nav left */}
+                <ul className="nav-main-left nav-main-ul hover:cursor-pointer">
+                    <li className="nav-main-li nav-section-one">
+                        <Link to="/" className="nav-main-item nav-item-home">
+                            TI <span className="text-color-highlight-orange">Social</span>
+                        </Link>
+                        {/* <i
+                            className="far fa-bars hamburger-menu"
+                            onClick={() => toggleHamburger()}
+                        ></i> */}
```
Comment (BelideNikhil) [inline src/components/NavBar/NavBar.jsx:16]: you can remove this I guess as it's a LInk now

**Review by BelideNikhil — APPROVED** [review]
UI looks nice, though it's not responsive yet.
Code and Folder structure are okay!
I'm curious on how you'll implement the user online/idle/offline feature

File: src/components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                        {!auth ? (
+                            <Link to="/login" className="nav-main-item nav-btn-login">
+                                Login
+                            </Link>
+                        ) : (
+                            <div className="nav-main-item nav-btn-login" onClick={signOutHandler}>
+                                Logout
+                            </div>
+                        )}
+                    </li> */}
```
Comment (vijaytembugade) [inline src/components/NavBar/NavBar.jsx:53]: Commented code.

File: src/components/NavBar/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                                Logout
+                            </div>
+                        )}
+                    </li>
+                    <li className="nav-main-li-res">
+                        <Link to="/" className="nav-right-btn" onClick={toggleHamburger}>
+                            Explore
+                        </Link>
+                    </li>
+                </ul> */}
```
Comment (vijaytembugade) [inline src/components/NavBar/NavBar.jsx:99]: Commented code.

**Review by vijaytembugade — APPROVED** [review]
Good use of tailwind and flowbite. Amazing UI. 🔥🔥

File: src/components/NavBar/NavBar.jsx (hunk 3 above)
Comment (tanveertkd) [reply src/components/NavBar/NavBar.jsx:99]: Will remove this. Still have a bit of work left with the navbar which is why I kept the commented code.

File: src/components/NavBar/NavBar.jsx (hunk 1 above)
Comment (tanveertkd) [reply src/components/NavBar/NavBar.jsx:16]: This will be the menu button once I make it responsive. Just commented it out for now

**Pull request merged**